### DISCUSSION TITLE
Auto start

### DISCUSSION THREAD

**Discussion by svetlinski** [netbox-community/netbox#8637] [Q&A]
Hi All! Apologies if this has been asked previously - I searched and I couldn't find anything similar in the Discussions.
I got a Netbox server running on Ubuntu, totally bare bones, but it works. But when I close my SSH session, I can no longer access it. I have to SSH back into the IP of the server, change my directory to /opt/netbox/netbox-3.1.6/netbox/ and issue the command "sudo python3 manage.py runserver 0.0.0.0:8000 --insecure" to get the web access to come back up.
I have followed instructions ([here](https://askubuntu.com/questions/1151080/how-do-i-run-a-script-as-sudo-at-boot-time-on-ubuntu-18-04-server) and [here](https://help.skysilk.com/support/solutions/articles/9000162390--basic-how-to-start-a-program-or-script-on-linux-automatically-on-boot-with-systemd#Create-the-sample-script-or-program-that-we-want-to-automatically-start-on-boot)), however, this has made no difference. As soon as I logout of the SSH session, the venv collapses, which I kinda understand - since there is no user session, the venv doesn't have anywhere to run.

How can I force it to continue running after I close the SSH connection? We have another user who needs access when I am not around and even if I can leave my SSH connection running, if my Internet goes down, Netbox will be down until I re-establish the SSH connection.

Thanks in advance.

**Comment by jeremystretch** [comment]
Please follow the [detailed installation documentation](https://netbox.readthedocs.io/en/stable/installation/) to install NetBox.

**Reply by svetlinski** [reply]
Thank you @jeremystretch for your reply. I have now fixed the issue, by continuing with the instructions. 
If anyone else comes across this post, please ensure that you do not stop at the venv part of the documentation. Carry on with the Gunicorn and HTTP/S server.
